Sources/SourceKitLSP/Swift/SwiftLanguageServer.swift

```diff
@@ -395,10 +393,6 @@ extension SwiftLanguageServer {
   }
 
   public func shutdown() async {
-    if let session = self.currentCompletionSession {
-      await session.close()
-      self.currentCompletionSession = nil
-    }
```

Comment:
I guess we no longer close on shutdown?

Reply:
Yes, that’s correct. Shouldn’t be an issue though. Unless you shut down the server, we keep the latest completion session alive anyway, so no memory concerns here and if you are shutting down sourcekit-lsp, then sourcekitd is about to be shut down as well, at which point it doesn’t matter if there’s still an open session since sourcekitd doesn’t do any kind of persisting when closing the session.